discover/reference/articles/knowledge-base-configuration.markdown

```diff
@@ -0,0 +1,683 @@
+   
+### Knowledge Base Section Configuration [](id=knowledge-base-section-configuration)
+
+-  **Show KBArticles sections title:** Sets whether to show the sections title
+   in the application. This is enabled by default.
+
+-  **kb articles sections:** Sets the section portlet's title with a comma 
+   comma-separated list. The default value is `general`.
+
+-  **kb article display style:** Sets the default kb article display style. 
+   Possible values are `title` or `abstract`. The default value is `title`.
+   
+-  **Show KBArticles pagination:** Sets whether to show pagination in the
+   application. This is enabled by default.
+   
+-  **Enable KBArticle description:** Sets whether to display the description
+   field in the edit view of the application. This is disabled by default.
+
+-  **Enable KBArticle ratings:** Sets whether to allow users to add ratings to
+   the Knowledge Base Article. This is enabled by default.
+
+-  **Show KBArticle attachments:** Sets whether to show the attachments for the 
+   Knowledge Base article. This is displayed by default.
+   
+-  **Show KBArticle asset entries:** Sets whether to show the asset entries for
+   the Knowledge Base Article. This is displayed by default.
+
+-  **Show KBArticle asset links:** Sets whether to show the asset links for the
+   Knowledge Base Article. This is displayed by default.
+
+-  **Enable KBArticle view count increment:** Sets whether to add a view counter 
+   to the Knowledge Base article. This is enabled by default.
+   
+-  **Enable KBArticle subscriptions:** Sets whether to allow user to subscribe
+   to the Knowledge Base article. This is enabled by default.
+
+-  **Enable KBArticle history:** Sets whether to allow users to view the history
+   for the Knowledge Base article. This is enabled by default.
+
+-  **Enable KBArticle print:** Sets whether to give users the option to print
+   the Knowledge Base article. This is enabled by default.
+   
+-  **Enable social bookmarks:** Sets whether to display social bookmarks 
+   when viewing a Knowledge Base article. This is disabled by default.
+   
+-  **Social bookmarks display style:** Sets the display style of the social
+   bookmarks. Possible values are `menu`, `simple`, `vertical`, or `horizontal`.
+   The default value is `menu`.
+
+-  **Social bookmarks display position:** Sets the position to display social 
+   bookmarks. Possible values are `top` or `bottom`. The default value is 
+   `bottom`.
+   
+-  **Social bookmarks types:** A comma-separated list of the possible bookmark 
+   types (twitter,facebook,plusone for example). Overwrite with a 
+   `portal-ext.properties` file. The default value is 
+   `${server-property://com.liferay.portal/social.bookmark.types}`.
+   
+-  **Admin KBArticle sections:** Sets the Knowledge Base article sections 
+   available to the portlet.
+   
+-  **Admin KBArticle sections default:** Sets the default selected section for
+   Knowledge Base articles.
+
```

Comment:
It would be best to add a transition here, before the next h2 section. We should inform the reader that we're "switching gears".